### PR TITLE
Document usePciIntake

### PR DESCRIPTION
**Pull request by RomanGaignault**
### What does this PR do? What is the motivation?
Document usePciIntake init parameter in [Browser Log Collection](https://docs.datadoghq.com/logs/log_collection/javascript/)

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/logs/log_collection/javascript.md [hunk 1]
```diff
@@ -375,6 +375,7 @@ Options that must have a matching configuration when using the `RUM` SDK:
 | `useSecureSessionCookie`               | Boolean | No       | `false` | Use a secure session cookie. This disables logs sent on insecure (non-HTTPS) connections.                                                                                |
 | `usePartitionedCrossSiteSessionCookie` | Boolean | No       | `false` | Use a partitioned secure cross-site session cookie. This allows the logs SDK to run when the site is loaded from another one (iframe). Implies `useSecureSessionCookie`. |
 | `useCrossSiteSessionCookie`            | Boolean | No       | `false` | **Deprecated**, see `usePartitionedCrossSiteSessionCookie`.                                                                                                              |
+| `usePciIntake`                         | Boolean | No       | `false` | Set to true to forward logs to the PCI-compliant intake. PCI-compliant intake is only available for Datadog organizations in the us1 site. If `usePciIntake` is set to true and site is not datadoghq.com, logs are sent to the default intake. |
```
Comment (BenoitZugmeyer) [inline content/en/logs/log_collection/javascript.md:378]: **💬 suggestion:** ‏
Maybe add a link to https://docs.datadoghq.com/data_security/logs/#pci-dss-compliance-for-log-management?

**Review by BenoitZugmeyer — COMMENTED** [review]
Looks great, thanks

File: content/en/logs/log_collection/javascript.md [hunk 2]
```diff
@@ -1261,3 +1262,4 @@ window.DD_LOGS && window.DD_LOGS.getInternalContext() // { session_id: "xxxx-xxx
 [13]: https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted
 [14]: /integrations/content_security_policy_logs/#use-csp-with-real-user-monitoring-and-session-replay
 [15]: #user-tracking-consent
+[16]: https://docs.datadoghq.com/data_security/logs/#pci-dss-compliance-for-log-management
```
Comment (BenoitZugmeyer) [inline content/en/logs/log_collection/javascript.md:1265]: **🥜 nitpick:** ‏I think we might want to use a relative link here? (Not sure if it matters)

```suggestion
[16]: /data_security/logs/#pci-dss-compliance-for-log-management
```

**Review by BenoitZugmeyer — APPROVED** [review]

File: content/en/logs/log_collection/javascript.md [hunk 3]
```diff
@@ -375,6 +375,7 @@ Options that must have a matching configuration when using the `RUM` SDK:
 | `useSecureSessionCookie`               | Boolean | No       | `false` | Use a secure session cookie. This disables logs sent on insecure (non-HTTPS) connections.                                                                                |
 | `usePartitionedCrossSiteSessionCookie` | Boolean | No       | `false` | Use a partitioned secure cross-site session cookie. This allows the logs SDK to run when the site is loaded from another one (iframe). Implies `useSecureSessionCookie`. |
 | `useCrossSiteSessionCookie`            | Boolean | No       | `false` | **Deprecated**, see `usePartitionedCrossSiteSessionCookie`.                                                                                                              |
+| `usePciIntake`                         | Boolean | No       | `false` | Set to true to forward logs to the [PCI-compliant intake][16]. PCI-compliant intake is only available for Datadog organizations in the us1 site. If `usePciIntake` is set to true and site is not datadoghq.com, logs are sent to the default intake. |
```
Comment (estherk15) [inline content/en/logs/log_collection/javascript.md:378]:
```suggestion
| `usePciIntake`                         | Boolean | No       | `false` | To forward logs to the [PCI-compliant intake][16], set to `true`. The PCI-compliant intake is only available for Datadog organizations in the US1 site. If `usePciIntake` is set to `true` and the site is not US1 (datadoghq.com), logs are sent to the default intake. |
```

**Review by estherk15 — APPROVED** [review]
Have one non-blocking suggestion, approved! Add the `/merge` comment to add to the merge queue!

**Pull request merged**